ui/src/tempVars/components/TemplateVariableEditor.tsx

```diff
@@ -291,7 +323,18 @@ class TemplateVariableEditor extends PureComponent<Props, State> {
       return
     }
     const {onUpdate, onCreate, notify} = this.props
-    const {nextTemplate, isNew} = this.state
+    const {
+      nextTemplate,
+      isNew,
+      isDynamicSourceSelected,
+      selectedSource,
+    } = this.state
+
+    nextTemplate.sourceID = '0'
```

Comment:
I'm assuming '0' is a source ID that can never exist?  Perhaps document this ID's purpose with a variable?

Reply:
'0' is the default / dynamic database - so this clearly needs a comment since your assumption is WRONG 😆